Localization/Resources/de-DE/winget.resw

```diff
@@ -202,7 +202,7 @@
     <comment>{Locked="{0}"} Error message displayed when the user provides an extra positional argument when none was expected. {0} is a placeholder replaced by the user's extra argument input.</comment>
   </data>
   <data name="FeatureDisabledMessage" xml:space="preserve">
-    <value>Diese Funktion ist zurzeit in Arbeit und kann hat in Zukunft geändert oder entfernt werden. Um es zu aktivieren, bearbeiten Sie Ihre Einstellungen („winget settings“), um die Experimentelle Funktion einzuschließen: „{0}“</value>
+    <value>Diese Funktion ist noch in Entwicklung und kann in Zukunft geändert oder entfernt werden. Um Sie zu aktivieren, bearbeiten Sie Ihre Einstellungen („winget settings“) und fügen Sie die experimentelle Funktion hinzu: „{0}“</value>
```

Comment:
`[...] Um Sie zu aktivieren [...]`
The context is the feature, not the user. So shouldn't it be lowercase `sie`?

Reply:
Good catch, thanks. Fixed it.